rasa/core/policies/keras_policy.py

```diff
@@ -223,23 +218,22 @@ def continue_training(
         batch_size = kwargs.get("batch_size", 5)
         epochs = kwargs.get("epochs", 50)
 
-        with self.graph.as_default(), self.session.as_default():
-            for _ in range(epochs):
-                training_data = self._training_data_for_continue_training(
-                    batch_size, training_trackers, domain
-                )
+        for _ in range(epochs):
+            training_data = self._training_data_for_continue_training(
+                batch_size, training_trackers, domain
+            )
 
-                # fit to one extra example using updated trackers
-                self.model.fit(
-                    training_data.X,
-                    training_data.y,
-                    epochs=self.current_epoch + 1,
-                    batch_size=len(training_data.y),
-                    verbose=obtain_verbosity(),
-                    initial_epoch=self.current_epoch,
-                )
+            # fit to one extra example using updated trackers
+            self.model.fit(
```

Comment:
This seems to repeat most of the `model.fit` code of above? Can we make this re-usable?

Reply:
what do you mean by "most"? calling `self.model.fit`? Everything else is different